CMakeLists.txt

```diff
@@ -1,24 +1,98 @@
 cmake_minimum_required(VERSION 3.10)
 project(lvgl)
 
-set(CMAKE_C_STANDARD 99)#C99 # lvgl officially support C99 and above
-set(CMAKE_CXX_STANDARD 17)#C17
+# Please set the wanted option to 'ON'. By default, FBDEV is used
+# be sure to also enable the selected driver in lv_conf.h
+
+option(LV_USE_WAYLAND "Use the wayland client backend" OFF)
```

Comment:
Hi, there is a potential issue here: [https://cmake.org/cmake/help/latest/command/option.html](https://cmake.org/cmake/help/latest/command/option.html)

![image](https://github.com/user-attachments/assets/dd71aad5-8868-4ec7-87c1-cd737007e53d)

Before that, it should be ensured that the cache has been cleared, e.g:

```shell
if(DEFINED LV_USE_WAYLAND)
        unset(LV_USE_WAYLAND CACHE)
endif()
```

Reply:
Hey @YobeZhou I am not quite well versed at CMake, I've copied what I found in the existing SDL simulator https://github.com/lvgl/lv_port_pc_vscode/blob/master/CMakeLists.txt 

I usually ``rm -rf build`` when I change options.